### ISSUE TITLE
Leading dots break linter

### ISSUE THREAD

**Issue by Scukerman** [uber/prototool#235]
Prototool linter breaks down when leading dots are used.
PoC is here: https://github.com/Scukerman/prototool-lint-bug

**Packages and Name Resolution**
https://developers.google.com/protocol-buffers/docs/proto3#packages-and-name-resolution

**Comment by peter-edge** [comment]
The Protobuf file:

```proto
syntax = "proto3";

package super.game;

option go_package = "gamepb";
option java_multiple_files = true;
option java_outer_classname = "GameProto";
option java_package = "com.super.game";

import "game/game.proto";

message Resource {
    .game.Resource game = 1;
}
```

The issue:

```
super/game/game.proto:13:5:Field name "" must be lower_snake_case.
```

This is almost for sure an issue with github.com/emicklei/proto - @Scukerman would you be able to file this over there, and potentially look into writing a fix?

In the future, please follow the Issue Guidelines and provide the exact replication of the problem in the GitHub issue. We appreciate the bug report however!

**Comment by peter-edge** [comment]
@amckinney @AlexAPB when https://github.com/emicklei/proto/pull/93 is merged, `glide.yaml` in Prototool will have to be updated to the updated commit.

**Comment by AlexAPB** [comment]
Fixed in uber/prototool/pull/236

This will be released in the next prototool release
